src/tests/functional/plugin/cpu/single_layer_tests/nonzero.cpp

```diff
@@ -175,6 +175,26 @@ std::vector<InputShape> inShapesDynamic = {
                 {4, 4, 4, 200},
                 {4, 4, 4, 300}
             }
+        },
+        {
+            // dynamic shape
+            {-1, -1, -1, -1, -1},
+            { // target static shapes
+                {4, 24, 5, 6, 4},
+                {8, 32, 9, 10, 2},
+                {4, 24, 5, 6, 4},
+                {16, 48, 9, 12, 2}
+            }
+        },
+        {   
```

Comment:
/agent/_work/1/openvino/src/tests/functional/plugin/cpu/single_layer_tests/nonzero.cpp:189:  Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]
Done processing /agent/_work/1/openvino/src/tests/functional/plugin/cpu/single_layer_tests/nonzero.cpp

Reply:
This cause cpplint failure in CI.